src/librustc_resolve/lib.rs

```diff
@@ -909,7 +935,11 @@ enum PathResult<'a> {
 }
 
 enum ModuleKind {
+    /// Inline `mod something { ... }`.
     Block(NodeId),
+    /// Module from another file.
+    ///
+    /// Also called a normal module in the following code.
```

Comment:
A normal module is a `mod` module (inline or not). `ModuleKind::Def` could also be a trait or an enum.